Formula/libfaketime.rb

```diff
@@ -26,8 +26,17 @@ def install
   end
 
   test do
-    cp "/bin/date", testpath/"date" # Work around SIP.
+    (testpath/"test.c").write <<~EOS
+      #include <stdio.h>
+      #include <time.h>
+
+      int main(void) {
+        printf("%d\\n",(int)time(NULL));
+        return 0;
+      }
+    EOS
+    system ENV.cc, "test.c", "-o", "test"
     assert_match "1230106542",
-      shell_output(%Q(TZ=UTC #{bin}/faketime -f "2008-12-24 08:15:42" #{testpath}/date +%s)).strip
+      shell_output(%Q(#{bin}/faketime -f "2008-12-24 08:15:42" ./test)).strip
```

Comment:
This looks short enough to be moved up into the `assert_match` line above instead of its own line below.

Also, we can probably make this
```suggestion
      shell_output("#{bin}/faketime -f '2008-12-24 08:15:42' ./test").strip
```

Reply:
Thanks, I have adjusted the PR with those changes now.